.changeset/tricky-spiders-matter.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/truncate': major
+'@twilio-paste/core': major
+---
+
+[Truncate] Added the title prop to show full non-truncated text
```

Comment:
Let's add something about this being a breaking change, and what teams will need to change.